complaints/templates/complaints/login.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>CCIT BPHC</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <link rel="stylesheet" href="main.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+
+<style>.navbar {border-radius:0px;}</style>
+
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+	  <img class="navbar-brand" src="bitslogo.png"></img>
+      <a class="navbar-brand" href="#" style="color:white;">CCIT BPHC</a>
+    </div>
+    <ul class="nav navbar-nav">
+      <li><a href="contact_before.html">Contact Us</a></li>
+    </ul>
+    <ul class="nav navbar-nav navbar-right">
+      <li class="active"><a href="login.html"><span class="glyphicon glyphicon-user"></span> Login</a></li>
+    </ul>
+  </div>
+</nav>
+
+<div class="container">
+
+  <div class="row" style="">
+    <div class="Absolute-Center is-Responsive">
+	<center><img src="ccit.png" alt="..." style="background:#EEEEEE;border:0px;" class="img-thumbnail"></center><br><br>
+      <div class="col-sm-12 col-md-10 col-md-offset-1">
+        <form action="" id="loginForm">
+          <div class="form-group input-group">
+            <span class="input-group-addon"><i class="glyphicon glyphicon-user"></i></span>
+            <input class="form-control" type="text" name='username' placeholder="username"/>          
+          </div>
+          <div class="form-group input-group">
+            <span class="input-group-addon"><i class="glyphicon glyphicon-lock"></i></span>
+            <input class="form-control" type="password" name='password' placeholder="password"/>     
+          </div>
+		  </div> 
+          <div class="form-group">
+            <center><button type="button" class="btn" style="width:50px;display:block;"><i class="fa fa-unlock-alt" style="font-size:18px"></i></button></center>
+          </div>
+        </form>        
+       
+    </div>    
+  </div>
+</div>
+
+</body>
+</html>
```

Comment:
What is this file for? When we already have a login.html in accounts folder.Remove this if it is a duplicate file.